dowhy/causal_estimator.py

```diff
@@ -41,14 +43,14 @@ def __init__(self, data, identified_estimand, treatment, outcome,
         :param outcome: name of the outcome variable
         :param control_value: Value of the treatment in the control group, for effect estimation.  If treatment is multi-variate, this can be a list.
         :param treatment_value: Value of the treatment in the treated group, for effect estimation. If treatment is multi-variate, this can be a list.
-        :param test_significance: whether to test significance
+        :param test_significance: Method for testing statistical significance. If testing is not required, its value is False (default). All DoWhy estimators support "bootstrap" as the method name. Each estimator can implement its own specific statistical significance methods.
         :param evaluate_effect_strength: (Experimental) whether to evaluate the strength of effect
-        :param confidence_intervals: (Experimental) Binary flag indicating whether the confidence intervals should be computed.
+        :param confidence_intervals: Method name for computing confidence intervals. If intervals are not required, its value is False (default).
```

Comment:
Shouldn't we also mention that we support bootstrap confidence intervals by default for all estimators?

Reply:
Good point. Added more details.